pegasus/sites.v3/code.org/public/athletes.haml

```diff
@@ -2,38 +2,42 @@
 title: "<%= I18n.t(:sports_page_title) %>"
 theme: responsive
 social:
+  "og:title": "<%= I18n.t(:sports_og_title) %>"
+  "og:description": "<%= I18n.t(:sports_og_description) %>"
+  "og:image" : "<%= 'https://code.org/images/social-media/' + (PageMode.get(request) == 'feature-athletes' ? 'athletes-thumbnail.jpg' : 'sports.jpg') %>"
+  "og:image:width": "<%= PageMode.get(request) == 'feature-athletes' ? '1920' : '1200' %>"
```

Comment:
Looking at how many times we check page mode, it might be more readable to have two separate blocks wrapped with if & else.  That said, if this is going away in a few days so that we only have one page mode, then no big deal to keep it like this.